travis/check-doctum-parse-errors.sh

```diff
@@ -16,19 +16,6 @@ NOTESTS=$3
 
 if [ "$NOTESTS" == '1' ]
 then
-	# Workarounds for
-	# https://github.com/fabpot/Sami/issues/116
-	# and
-	# https://github.com/fabpot/Sami/issues/117
-	errors=$(
-		unbuffer phpBB/vendor/bin/sami.php parse build/sami-checkout.conf.php -v | \
-		sed "s,\x1B\[[0-9;]*[a-zA-Z],,g" | \
-		grep "ERROR: " | \
-		tee /dev/tty | \
-		wc -l
-	)
-	if [ "$errors" != "0" ]
-	then
-		exit 1
-	fi
+	(cd phpBB/vendor/bin/ && curl -O https://doctum.long-term.support/releases/latest/doctum.phar && chmod +x doctum.phar)
```

Comment:
I'd suggest to put `doctum.phar` to the `build` folder permanently so it could be used not in tests only but also to create docs manually via command line.
Also `build/build.xml` still contains Sami references: https://github.com/code-lts/phpbb/blob/625a16732e47ffa858d804a9bb61a659e162df85/build/build.xml#L131
https://github.com/code-lts/phpbb/blob/625a16732e47ffa858d804a9bb61a659e162df85/build/build.xml#L137

cba467829b0d6af5dec5ba2ca3718f8ede8d6ab4 is already known and would help probably :)

Reply:
> Also build/build.xml still contains Sami references

:facepalm: By bad, vscode ignored pattern..

All fixed, thank you for the initial review